pkg/serverless/trace/span_modifer_test.go

```diff
@@ -23,10 +24,32 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/trace/info"
 	"github.com/DataDog/datadog-agent/pkg/trace/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/trace/testutil"
+	"github.com/DataDog/datadog-agent/pkg/trace/writer"
 
 	"github.com/DataDog/datadog-go/v5/statsd"
 )
 
+type mockTraceWriter struct {
+	mu       sync.Mutex
+	payloads []*writer.SampledChunks
+}
+
+func (m *mockTraceWriter) Stop() {
+	//TODO implement me
```

Comment:
Is there anything to actually implement here?
I think we can leave them as is until some test actually needs this to do something.